### ISSUE TITLE
Unable to deploy ingresses from Hydra chart on Kubernetes v1.19

### ISSUE THREAD

**Issue by EmmEff** [ory/k8s#171]
**Describe the bug**

Kubernetes 1.19 enforces the Ingress resource apiVersion (minimum `networking.k8s.io/v1beta1`)

**To Reproduce**

Run `helm install -g ory/hydra ...` on Kubernetes v1.19.x cluster.

**Expected behavior**

Successful deployment of ingress(es)

**Environment**

- latest Hydra Helm chart
- Kubernetes v1.19 cluster
- Helm v3.3.3

**Proposal**

Use boilerplate `apiVersion` declaration code from chart generated with `helm create`.

```yaml
{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
apiVersion: networking.k8s.io/v1beta1
{{- else -}}
apiVersion: extensions/v1beta1
{{- end }}
```

**Comment by jfelten** [comment]
strange I just installed with the same versions as you with no issues: I just opened https://github.com/ory/k8s/pull/172 to update the hydra version but didn't update any of the schemas.
''' helm version
version.BuildInfo{Version:"v3.3.3", GitCommit:"55e3ca022e40fe200fbc855938995f40b2a68ce0", GitTreeState:"clean", GoVersion:"go1.14.9"}
kubectl version
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.2", GitCommit:"f5743093fd1c663cb0cbc89748f730662345d44d", GitTreeState:"clean", BuildDate:"2020-09-16T13:41:02Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.0", GitCommit:"e19964183377d0ec2052d1f1fa930c4d7575bd50", GitTreeState:"clean", BuildDate:"2020-08-26T14:23:04Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by jfelten** [comment]
added your suggestion to my PR https://github.com/ory/k8s/pull/172

**Comment by EmmEff** [comment]
I am seeing the following message from `helm upgrade`:

```sh
Error: UPGRADE FAILED: cannot patch "se-hydra-public" with kind Ingress: admission webhook "validate.nginx.ingress.kubernetes.io" denied the request: rejecting admission review because the request does not contains an Ingress resource but networking.k8s.io/v1, Resource=ingresses with name se-hydra-public in namespace default
```

Admittedly, I am no K8s expert, so I might've misinterpreted the error message...

**Comment by aeneasr** [comment]
That will be fixed once #172 is merged

**Comment by aeneasr** [comment]
Resolved on master
